website/events/admin/event.py

```diff
@@ -252,3 +223,14 @@ def get_urls(self):
             ),
         ]
         return custom_urls + urls
+
+    def get_field_queryset(self, db, db_field, request):
+        """Members without the can view as organiser permission can only assign their own groups as organiser"""
+        pk = resolve(request.path_info).kwargs["object_id"]
+        if db_field.name == "organisers" and not request.user.has_perm(
+            "override_organiser"
```

Comment:
```suggestion
            "events.override_organiser"
```
based on other code this is currently incorrect